cdap-docs/reference-manual/source/http-restful-api/namespace.rst

```diff
@@ -217,3 +215,34 @@ namespace is retained, as it is always available in CDAP.
 
 To delete only the datasets of a namespace, there is a 
 :ref:`specific dataset endpoint <http-restful-api-dataset-deleting-all>` for that.
+
+.. _http-restful-api-namespace-configs:
+
+Namespace Configurations
+========================
+
+When creating or editing a namespace, namespace configurations can be supplied as a part of the
+request body. These would be in the form of a JSON string of configuration key-value pairs. Note
+that the values for these keys can not be modified once the namespace is created, with the exception
+of ``scheduler.queue.name``.
+
+.. rubric:: Namespace Configurations
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Configuration Name
+     - Description
+   * - ``scheduler.queue.name``
+     - Scheduler queue for CDAP programs and CDAP explore queries run in the namespace. See
```

Comment:
in the other documentation like this, we don't specify YARN (it just remains generic).